RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -68,22 +71,28 @@ void HLTPFDiJetCorrCheckerWithDiTau::produce(edm::StreamID iSId, edm::Event& iEv
 
   std::set<unsigned int> indices;
 
-  if (pfJets.size() > 1 && taus.size() > 1) {
-    for (unsigned int iJet1 = 0; iJet1 < pfJets.size(); iJet1++) {
-      for (unsigned int iJet2 = iJet1 + 1; iJet2 < pfJets.size(); iJet2++) {
-        bool correctComb = false;
-        const reco::PFJet& myPFJet1 = pfJets[iJet1];
-        const reco::PFJet& myPFJet2 = pfJets[iJet2];
+  for (unsigned int iJet1 = 0; iJet1 < pfJets.size(); iJet1++) {
+    for (unsigned int iJet2 = iJet1 + 1; iJet2 < pfJets.size(); iJet2++) {
+      bool correctComb = false;
+      const reco::PFJet& myPFJet1 = pfJets[iJet1];
+      const reco::PFJet& myPFJet2 = pfJets[iJet2];
+
+      if ((myPFJet1.p4() + myPFJet2.p4()).M() < mjjMin_)
+        continue;
 
-        if ((myPFJet1.p4() + myPFJet2.p4()).M() < mjjMin_)
+      for (unsigned int iTau1 = 0; iTau1 < taus.size(); iTau1++) {
+        if (reco::deltaR2(taus[iTau1]->p4(), myPFJet1.p4()) < dRmin2_)
+          continue;
+        if (reco::deltaR2(taus[iTau1]->p4(), myPFJet2.p4()) < dRmin2_)
           continue;
 
-        for (unsigned int iTau1 = 0; iTau1 < taus.size(); iTau1++) {
-          if (reco::deltaR2(taus[iTau1]->p4(), myPFJet1.p4()) < dRmin2_)
-            continue;
-          if (reco::deltaR2(taus[iTau1]->p4(), myPFJet2.p4()) < dRmin2_)
+        if (taus.size() == 1) {
+          if (taus[iTau1]->pt() < extraTauPtCut_)
             continue;
 
+          correctComb = true;
+          break;
```

Comment:
```suggestion
```
The `break` here can be removed, as it's sort-of redundant with the one in L110.

Reply:
Right. I'll also remove the ```break``` on line 106 for the same reason. Both cases are caught by the lines 109 and 110.

Reply:
Mh.. the one on L106 is needed to exit the 2nd loop on the tau candidates (which is only in the `else` block and not in the `if` block); I think it makes sense to keep it (i.e. the one on L106).